docs/elements/hx-file-tile/index.html

```diff
@@ -99,14 +112,29 @@ <h2 id="methods">Methods</h2>
       Manipulates the <code>icon</code> attribute.
     </dd>
 
+    <dt>invalid {Boolean}</dt>
+    <dd>
+      Manipulates the <code>invalid</code> attribute.
+    </dd>
+
     <dt>loading {Boolean}</dt>
     <dd>
       Manipulates the <code>loading</code> attribute.
```

Comment:
`Returns <code>true</code> when the <code>progress</code> attribute is present.`